ohif-configurable/config-mode/src/initToolGroups.js

```diff
@@ -53,6 +53,7 @@ function initDefaultToolGroup(
       { toolName: toolNames.Angle },
       { toolName: toolNames.Magnify },
       { toolName: toolNames.SegmentationDisplay },
+      { toolName: toolNames.CalibrationLine },
```

Comment:
This had better go in a custom mode unless the calibration stuff has been merged to v3-stable/main for OHIF/CS3D.